riscv/processor.h

```diff
@@ -304,6 +312,10 @@ class processor_t : public abstract_device_t
   bool check_triggers_icount;
   std::vector<bool> impl_table;
 
+  std::bitset<NUM_ISA_EXTENSIONS> extension_enable_table;
```

Comment:
I would add a comment here: "Note: does not include single-letter extensions in misa"